### PR TITLE
Improved readme and website, please to #34.

### PR DESCRIPTION
**Pull request by zrzjohn**
@KevinZonda 
### Create English & CHT Website

添加了网站的英文页面和繁体中文页面，并改善了您的网站页面（具体请您浏览[ zrzjohn / LGBTQIA-in-China ](https://github.com/zrzjohn/LGBTQIA-in-China)）。

### Improved Readme

添加了英文和繁体中文的Readme，并改善了语言选择菜单。

### CODE REVIEW

File: README.ZH.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+## 參與
+
+1. 在 README.md 中署名 ([署名](#署名))
+2. 寫篇關於中國 LGBTQIA 群體的文章（[文章](#文章)）
+
+## 署名
+
+> 這將造成壹定程度上的隱私泄漏，因為在中國坦白自己的性取向、性別認同及性別表達仍是個風險較大的行為。
+
+你可以查看[简体中文 Readme ](README.md)中的支持者名单。
```
Comment (KevinZonda) [inline README.ZH.md:24]: If you want to add spaces, add them beside the hyperlink.

File: english.html [hunk 2]
```diff
@@ -0,0 +1,400 @@
...
+		content="LGBTQIA LGBT Homosexual gay lesbian transgender intersexual asexual bisexual queer questioning 同性恋 跨性别 双性人 酷儿 双性恋 疑性恋">
+	<title>LGBTQIA in China</title>
+	<link
+		href="https://fonts.googleapis.com/css?family=Noto+Sans+SC:300|Noto+Serif+SC:300|ZCOOL+KuaiLe|ZCOOL+XiaoWei&display=swap"
+		rel="stylesheet">
+	<link rel="stylesheet" href="https://cdn.jsdelivr.net/gh/twbs/bootstrap/dist/css/bootstrap-grid.min.css">
+	<link rel="stylesheet" href="https://cdn.jsdelivr.net/gh/alvarotrigo/fullPage.js/dist/fullpage.min.css">
+	<link rel="stylesheet" href="https://at.alicdn.com/t/font_1640253_7ac8t2q6btg.css">
+	<link rel="stylesheet" href="https://at.alicdn.com/t/font_1912611_olza2ei6vv.css">
+	<style>
```
Comment (KevinZonda) [inline english.html:24]: I think we should peel CSS out...

File: README.ZH.md [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+
+## 文章
+
+> 建議每篇文章均在 `/post` 下提供 .md 格式的備份。
+>
+> 這些是支持本項目的人貢獻的文章。
+
+| 標題 | 作者 | 原文 | 備份 |
```
Comment (KevinZonda) [inline README.ZH.md:32]: In my view, we should maintainer only articals in zh-hans's readme.

Comment (zrzjohn) [reply README.ZH.md:32]: I think we should use it SIGN.md replace `## 署名`

Comment (scvoet) [reply README.ZH.md:32]: > ## 署名

That  sounds bad. Put it in readme.md is better to show who supported our project.

**Comment by zrzjohn** [comment]
@KevinZonda @scvoet 刚才我的仓库出问题了，我重新 Fork 了一下。新的拉取请求看[这里](https://github.com/LGBT-CN/LGBTQIA-in-China/pull/34)。

**Pull request closed without merging**